faqs.rst

```diff
@@ -20,3 +20,82 @@ How do I report a bug?
 ----------------------
 
 You can read up on reporting bugs here: :ref:`contributing` or report one directly for flux `core <https://github.com/flux-framework/flux-core/issues>`_ or `sched <https://github.com/flux-framework/flux-sched/issues>`_.
+
+.. _not_managing_all_resources:
+
+---------------------------------------------------------------------------------
+Why is Flux not discovering and managing all of the resources on the system/node?
+---------------------------------------------------------------------------------
+
+* Various bind flags passes to the parallel launcher that started Flux
+* CUDA enabled hwloc (for GPUs)
+
+.. _launch_large_num_jobs:
+
+------------------------------------------------------------
+How do I efficiently launch a large number of jobs to Flux?
+------------------------------------------------------------
+
+See `bulksumbission <https://github.com/flux-framework/flux-workflow-examples/tree/master/async-bulk-job-submit>`_ for an example workflow or flux `tree <https://github.com/flux-framework/flux-sched/blob/master/src/cmd/flux-tree>`_.
+
+.. _node_memory_exhaustion:
+
+------------------------------------------------------------------
+Memory exhaustion on a node when running large ensembles with Flux
+------------------------------------------------------------------
+
+Set ``rundir`` so that the content store is not saved to ``/tmp`` but to filesystem.
```

Comment:
I would add an example CLI invocation that sets the rundir.  For example: `flux start -o,-Srundir=/path/to/rundir`.